index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/bf723a76a7.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="Css/style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav class="mobile">
+            <a href="#">
+            <div class="hamburger">
+                <span class="bars"></span>
+                <span class="bars"></span>
+                <span class="bars"></span>
+            </div>
+            </a>
+            <ul class="disappear" id="navList">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About Us</a> </li>
```

Comment:
- [ ] [OPTIONAL] It would be nice to remove the home page link when you are already on the home page.